keps/NNNN-kep-template/README.md

```diff
@@ -445,6 +461,8 @@ The KEP must have a approver from the PRR review team.
 
 ### Scalability
 
+This section must be completed when targeting beta graduation to a release.
```

Comment:
I would like this to be completed in Alpha, to force people think about it. People often don't think about scalabilty, so that would force them to think if their design scales.

I agree that even longer-term probably approval just at Beta is good enough.


BTW - I started thinking, whether we shouldn't use the same strategy for upgrade/rollout/rollback.

[The other that are purely production-oriented, is probably fine just to not force people to not think for Alpha].

@deads2k @johnbelamaric - thoughts?

Reply:
The wording - to be cleanup I had on my mind is along the lines with:
"the section should be analyzed for Alpha to reduce risk of full redesigns for Beta, but will not require formal aproval until targeting Beta"

Reply:
> "the section should be analyzed for Alpha to reduce risk of full redesigns for Beta, but will not require formal aproval until targeting Beta"

For alpha, this section is encouraged: reviewers should consider these questions and attempt to answer them.
For beta, this section is required: reviewers must answer these questions.
For GA, this section is required: approvers should be able to confirms the previous answers based on experience in the field.